ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -0,0 +1,196 @@
+      delay: 2
+
+    - debug: msg="First Iteration flap all interfaces one by one on DUT"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - debug: msg="{{ device_conn }}"
+
+    - name: include continuous_link_flap_helper.yml
+      include: continuous_link_flap/continuous_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Second Iteration flap all interfaces one by one on DUT"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - name: include continuous_link_flap_helper.yml
+      include: continuous_link_flap/continuous_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Third Iteration flap all interfaces one by one DUT"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - name: include continuous_link_flap_helper.yml
+      include: continuous_link_flap/continuous_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="First Iteration flap all interfaces one by one on Peer Device"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - debug: msg="{{ device_conn }}"
+
+    - set_fact:
+        neighbors: "{{device_conn}}"
+
+    - name: include continuous_peer_link_flap_helper.yml
+      include: continuous_link_flap/continuous_peer_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Second Iteration flap all interfaces one by one on Peer Device"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - set_fact:
+        neighbors: "{{device_conn}}"
+
+    - name: include continuous_peer_link_flap_helper.yml
+      include: continuous_link_flap/continuous_peer_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - debug: msg="Third Iteration flap all interfaces one by one on Peer Device"
+
+    - name: Gathering lab graph facts about the device
+      conn_graph_facts: host={{ inventory_hostname }}
+      connection: local
+      tags: always
+
+    - set_fact:
+        neighbors: "{{device_conn}}"
+
+    - name: include continuous_peer_link_flap_helper.yml
+      include: continuous_link_flap/continuous_peer_link_flap_helper.yml interface="{{item}}"
+      with_items: "{{ device_conn.keys() }}"
+
+    - name: wait 60 secs so that BGP routes are relearned
+      pause: seconds=60
+
+    - name: Record  memory status at end
```

Comment:
>  [](start = 19, length = 1)

the same.

Reply:
Still there

---
In reply to: [274190739](https://github.com/Azure/sonic-mgmt/pull/810#discussion_r274190739) [](ancestors = 274190739)

Reply:
@qiluo-msft :  Not present in the latest commit. This is first diff. Thx.